scripts/aura.py

```diff
@@ -0,0 +1,82 @@
+"""
+A script to calculate % portions of user balance per given snapshot
+for FBVECVX, Rari and BVECVX LP Pool
+"""
+
+import math
+from decimal import Decimal
+from typing import Dict
+
+import simplejson as json
+from rich.console import Console
+
+from config.constants import addresses
+from helpers.enums import Network
+from rewards.boost.boost_utils import get_bvecvx_lp_ppfs
+from rewards.boost.boost_utils import get_bvecvx_lp_ratio
+from rewards.classes.Snapshot import Snapshot
+from rewards.snapshot.chain_snapshot import sett_snapshot
+from rewards.snapshot.claims_snapshot import claims_snapshot
+from rewards.snapshot.token_snapshot import fuse_snapshot_of_token
+
+console = Console()
+
+
+STARTING_BLOCK = 14848251
```

Comment:
Maybe rename `AURA_SNAPSHOT_BLOCK` for clarity

Reply:
Done